tests/Feature/Api/Administration/RolesTest.php

```diff
@@ -234,8 +234,55 @@ public function testSuccessfulRoleCreate()
         $transformed = (new RoleTransformer())->transform($role);
         $response->assertJson($transformed);
     }
-}
 
+    public function testEditRoleSuccess() 
+    {
+        $user = factory(User::class)->create([
+            'password' => Hash::make('password'),
+            'role_id'     => Role::where('code', Role::PROCESSMAKER_ADMIN)->first()->id,
+        ]);
+        $this->auth($user->username, 'password');
+        //create a role, but with a fixed name with a factory method 
+        $role = factory(\ProcessMaker\Model\Role::class)->create([
+            'code' => 'TESTROLE',
+            'uid' => '134A',
```

Comment:
Use the faker classes instead of coding the values.

https://github.com/fzaninotto/Faker